bazel/MODULE.bazel

```diff
@@ -0,0 +1 @@
+bazel_dep(name = "platforms", version = "0.0.9")
```

Comment:
To properly Bzlmod-ify this so that other Bazel projects can depend on it, you also need to add something like this to the top:

```starlark
module(
    name = "emsdk",
    version = "<current version>",
)
```

Instead of manually maintaining `<current version>`, you can also use the [Publish to BCR app](https://github.com/bazel-contrib/publish-to-bcr) to automate the version bump and releases.

Reply:
I hope I understand what you mean xD (I'm still new in bazel).  
If it's not a problem, I'll try to do it in the next/separate MR.  
Thanks for the tip!